src/Sim.h

```diff
@@ -0,0 +1,31 @@
+#include "Arduino.h"
```

Comment:
nit: it's may be better to put the includes within the include guards.

Reply:
Done!